src/dev/series.clj

```diff
@@ -1,5 +1,5 @@
 (ns series
-  (:require [java-time       :as t]
+  (:require [java-time.api   :as t]
```

Comment:
I'm getting deja vu, thought we already did this?

Reply:
I thought so too, but I guess I missed a spot.